source/behavior/blocks/format/components/geometry.json

```diff
@@ -4,18 +4,8 @@
   "description": "The description identifier of the geometry file to use to render this block. This identifier must match an existing geometry identifier in any of the currently loaded resource packs.",
   "oneOf": [
     {
-      "type": "string"
-    },
-    {
-      "type": "object",
-      "properties": {
-        "identifier": {
-          "title": "Identifier",
-          "description": "The description identifier of the geometry file to use to render this block. This identifier must match an existing geometry identifier in any of the currently loaded resource packs.",
-          "type": "string",
-          "enum": ["minecraft:geometry.full_block", "minecraft:geometry.cross"]
-        }
-      }
+      "type": "string",
+      "enum":  ["minecraft:geometry.full_block", "minecraft:geometry.cross"]
```

Comment:
Are they really enum, or can people use their own geometry 🤔 

Think it might be better to use a pattern matching with regex in the case that its not an enum

Reply:
People can use their own geometry. I'd just copied what was already there, from stirante I believe, and moved it.

Reply:
Since the enum isn't restricted to just those values I figured it was fine and it was nice to have those show up in autocompletes

Reply:
```json
"oneOf": [
        {
          "enum": [
            "minecraft:geometry.full_block",
            "minecraft:geometry.cross"
          ]
        },
        {
          "pattern": "minecraft:geometry\\.(full_block|cross)|geometry\\.[a-zA-Z0-9_]+"
        }
      ]
      ```
Do you think something like this would be good?

Reply:
Enum are restricted to those values, so they if someone specifies something else it would error 🤔 

Might have to revisit some other parts then later.

think a `[a-zA-Z0-9_\.:\=]` would be good enough 😄